import-service/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "import-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
+  "main": "handler.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "aws-sdk": "^2.792.0",
```

Comment:
shouldn't sdk be in devDependencies? lambda has its own sdk instance and node_modules doesn't need to have this package